gdal/doc/source/drivers/vector/index.rst

```diff
@@ -83,6 +83,7 @@ Vector drivers
    ntf
    oapif
    oci
+   oda
```

Comment:
I wouldn't list this here, as it will appear as a driver. I believe you'll need to include it in a hidden toctree from dwg.rst or dgnv8.rst (probably not both)
Something like the following added at the end of the page:

```
.. toctree::
   :maxdepth: 1
   :hidden:

   oda
```

Reply:
Done!